vhdbuilder/packer/test/linux-vhd-content-test.sh

```diff
@@ -65,7 +65,18 @@ testImagesPulled() {
 
```

Comment:
(In general) would `aks-rp` need changes to specify if the version to be used is part of `multiArchVersions` or `amd64OnlyVersion`? I presume that `aks-rp` does not need that but I am not sure.

Reply:
aks-rp doesn't need changes to specify if the version to be used is part of multiArchVersions or amd64OnlyVersion. But eventually after all images has 'multi-arch' version, all images tag used by RP for k8s 1.22+ (we will support arm64 nodes on AKS 1.22+ cluster) need to be 'multi-arch'. I will (or work with aks partner team) to switch to 'multi-arch' in following RP changes in order that AKS can support amd64/arm64 nodes in the same cluster.